woodwork/table_accessor.py

```diff
@@ -708,7 +708,7 @@ def mutual_information(self, num_bins=10, nrows=None, include_index=False):
         """Calculates mutual information between all pairs of columns in the DataFrame that
         support mutual information. Logical Types that support mutual information are as
         follows:  Boolean, Categorical, CountryCode, Datetime, Double, Integer, Ordinal,
-        SubRegionCode, and ZIPCode
+        SubRegionCode, and PostalCode
```

Comment:
here too (maybe it's a little silly that we have this docstring in three separate places)